editor/filesystem_dock.cpp

```diff
@@ -944,6 +944,21 @@ void FileSystemDock::_select_file(const String &p_path, bool p_select_in_favorit
 			fpath = fpath.substr(0, fpath.length() - 1);
 		}
 	} else if (fpath != "Favorites") {
+		if (FileAccess::exists(fpath + ".import")) {
+			Ref<ConfigFile> config;
+			config.instance();
+			Error err = config->load(fpath + ".import");
+			if (err == OK) {
+				if (config->has_section_key("remap", "importer")) {
+					String importer = config->get_value("remap", "importer");
+					if (importer == "keep") {
+						EditorNode::get_singleton()->show_warning(TTR("Importing has been disabled for this file, so it can't be opened."));
```

Comment:
It's still possible to open the file via scripting, so I think this makes it more clear:

```suggestion
						EditorNode::get_singleton()->show_warning(TTR("Importing has been disabled for this file, so it can't be opened in the editor."));
```

Reply:
maybe it should be renamed from opened to edited instead?

Reply:
changed